### DISCUSSION TITLE
reconnect after dreame account password change

### DISCUSSION THREAD

**Discussion by zhenyatk** [Tasshack/dreame-vacuum#549] [Q&A] [answered]
Hello
I'm change dreame account password, and now integration can't connect to device. Is it possible reauthorize integration, or only delete and setup new now?

**Comment by Tasshack** [comment, marked as answer]
Try using the `add device` button again with your new credentials. Integration should update your password event it is going to say `the device is already configured`. Also please not that integration will try to connect using your old password periodically in the background and that could lead server blocking your ip so you may also need to restart your router.
